Makefile

```diff
@@ -59,9 +59,7 @@ delete_minikube:
 copy_terraform_files:
 	mkdir -p build/terraform
 	FILE=build/terraform/terraform.tfvars.json
-	@if [ ! -f "build/terraform/terraform.tfvars.json" ]; then\
-		cp -r terraform_files/* build/terraform/;\
-	fi
+	cp -r terraform_files/* build/terraform/;\
```

Comment:
not sure even that this target still needed
it will not help, 
look on https://github.com/tsorya/test-infra/pull/17/files#diff-a39f09607192a22eb18fce5fcc2378e5L27